readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -165,7 +149,7 @@ def append_conf(self, **__):
 
         extra_javascript_list = [
             'readthedocs-data.js',
-            '%score/js/readthedocs-doc-embed.js' % static_url,
+            '%sjavascript/readthedocs-doc-embed.js' % static_url,
```

Comment:
We are using these URLs from sphinx

https://github.com/readthedocs/readthedocs-sphinx-ext/blob/ea4643d0c204756eefcd32a45b75795686795daf/readthedocs_ext/readthedocs.py#L159-L161

core/js/ and javscript/ point to the same resource